extensions/mapreduce/src/main/java/org/radargun/traits/MapReducer.java

```diff
@@ -74,7 +48,7 @@
        *           parameters for the Mapper object
        * @return this builder instance
        */
-      Builder<KOut, VOut, R> mapper(String mapperFqn, Map<String, String> mapperParameters);
+      Builder<KOut, VOut, R> mapper(String mapperFqn, Collection<KeyValueProperty> mapperParameters);
```

Comment:
MapReduceStage uses `KeyValueProperty` type to specify the mapper/reducer/... params. No need to convert this twice.

https://github.com/radargun/radargun/pull/360/files#diff-faaddb54bc834e3a91fddc93c42acba1R61